duckdns/data/run.sh

```diff
@@ -83,15 +83,23 @@ while true; do
 
     if [[ ${ipv6} == *:* ]]; then
         if answer="$(curl -s "https://www.duckdns.org/update?domains=${DOMAINS}&token=${TOKEN}&ipv6=${ipv6}&verbose=true")" && [ "${answer}" != 'KO' ]; then
-            bashio::log.info "${answer}"
+            if [[ "${answer}" == *UPDATED* ]]; then
+		bashio::log.info "${answer}"
+	    else
+   		bashio::log.debug "${answer}"	  			
```

Comment:
I think I rather prefer the opposite way, as you wrote the description:

```suggestion
            if [[ "${answer}" == *NOCHANGE* ]]; then
   		bashio::log.debug "${answer}"
	    else
		bashio::log.info "${answer}"
```